src/streaming/controllers/ProtectionController.js

```diff
@@ -439,20 +496,26 @@ MediaPlayer.dependencies.ProtectionController = function () {
             this[MediaPlayer.models.ProtectionModel.eventList.ENAME_KEY_STATUSES_CHANGED] = onKeyStatusesChanged.bind(this);
 
             keySystems = this.protectionExt.getKeySystems();
-            this.protectionModel = this.system.getObject("protectionModel");
-            this.protectionModel.init();
+
+            // For browsers that don't support any version of EME, the app crashes without this check
+            if (this.system.hasMapping("protectionModel")) {
+                this.protectionModel = this.system.getObject("protectionModel");
+                this.protectionModel.init();
+            }
 
```

Comment:
@rashmimittal I'd like to do this check outside of ProtectionController.  If the browser does not support EME, We shouldn't be instantiating any protection objects.  This check should be done in Stream.js, I believe.  I'll add a comment where I think it should be.